src/actionPanel/DefaultActionPanel.tsx

```diff
@@ -39,7 +38,11 @@ const OnboardingContent: React.FunctionComponent = () => (
           </span>{" "}
           Activate a pre-made template from the Templates page.
         </p>
-        <Button onClick={async () => openExtensionOptions()} variant="info">
+        <Button
+          href="/options.html#/templates"
+          target="_blank"
```

Comment:
Codacy has a fix for the issue: [Functions that return promises must be async.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7805688)
```suggestion
          <Button onClick={async () => openExtensionOptions()} variant="info">
```